log/command.go

```diff
@@ -147,3 +146,33 @@ var levelToLog = map[string]log.Level{
 	"error": log.ErrorLevel,
 	"fatal": log.FatalLevel,
 }
+
+func (o Options) logToStyle(level log.Level, defaults *log.Styles) lipgloss.Style {
+	var levelStyle = style.Styles{}
+	switch level {
+	case log.Level(math.MaxInt32):
+	case log.DebugLevel:
+		levelStyle = o.LevelDebugStyle
+	case log.InfoLevel:
+		levelStyle = o.LevelInfoStyle
+	case log.WarnLevel:
+		levelStyle = o.LevelWarnStyle
+	case log.ErrorLevel:
+		levelStyle = o.LevelErrorStyle
+	case log.FatalLevel:
+		levelStyle = o.LevelFatalStyle
+	}
+
+	// NB: Empirically determined that _this_ is what we get if we pass no CLI/Env args
+	var defaultLevelStyle = style.Styles{Align: "left", Bold: true, Border: "none", Margin: "0 0", Padding: "0 0"}
+	if reflect.DeepEqual(levelStyle, defaultLevelStyle) {
```

Comment:
If there's a neater way to check "has the user set any `level.$level.` option?", I'm all ears :)